core/common.ts

```diff
@@ -310,4 +312,31 @@ export function defineBlocks(blocks: {[key: string]: BlockDefinition}) {
   }
 }
 
+/**
+ * Handle a key-down on SVG drawing surface. Does nothing if the main workspace
+ * is not visible.
+ *
+ * @internal
+ * @param e Key down event.
+ */
+// TODO (https://github.com/google/blockly/issues/1998) handle cases where there
```

Comment:
This appears out of date. Is it still relevant? If so, perhaps we should file a new issue and update this accordingly.